drivers/tapdisk-disktype.c

```diff
@@ -159,7 +153,6 @@ const disk_info_t *tapdisk_disk_types[] = {
 	[DISK_TYPE_LLPCACHE]    = &llpcache_disk,
 	[DISK_TYPE_LLECACHE]    = &llecache_disk,
 	[DISK_TYPE_NBD]         = &nbd_disk,
-	[DISK_TYPE_NTNX]        = &ntnx_disk,
```

Comment:
Should the `DISK_TYPE_NTNX` define be removed as well?

Reply:
yes, yes it should, thanks.